src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -15,6 +15,14 @@
   private int popularity;
   private long id;
 
+  // Recipe constructor for the recipe cards for which less information is needed
+  public Recipe(long id, String name, String imageBlobKey, String description) {
```

Comment:
You also may want to initialize the array lists in this constructor (just to empty lists). Even though you won't be using them, it's a good idea to have all your class variables initialized so you don't run into null pointer exceptions later on